### PR TITLE
Fix the examples.

### PR DESCRIPTION
**Pull request by mikebolt**
Finally!

### CODE REVIEW

**Comment by trusktr** [comment]
Changes LGTM. Anything else or is that all that is needed?

**Comment by mikebolt** [comment]
That’s it!

File: examples/01_bars.html [hunk 1]
```diff
@@ -48,8 +48,8 @@
 
 					var elem = { x: startValue, domElement: domElement };
 
-					var updateCallback = function() {
-						this.domElement.style.left = this.x + 'px';
+					var updateCallback = function(object) {
+						object.domElement.style.left = this.x + 'px';
```
Comment (martikaljuve) [inline examples/01_bars.html]: `this.x` needs to be changed to `object.x`.

File: examples/05_video_and_time.html [hunk 2]
```diff
@@ -40,9 +40,9 @@
 					new TWEEN.Tween( { x: 50, y: 0 } )
 						.to( { x: 400 }, this.duration )
```
Comment (martikaljuve) [inline examples/05_video_and_time.html]: I'm getting `NaN` for `this.duration` when testing this example locally. The video metadata might not be loaded yet when the 'play' event is triggered. Maybe we should use 'playing' event instead.

File: examples/01_bars.html (hunk 1 above)
Comment (mikebolt) [reply examples/01_bars.html]: Fixed

File: examples/05_video_and_time.html (hunk 2 above)
Comment (mikebolt) [reply examples/05_video_and_time.html]: I don't have this problem. Could you create a separate issue for this? I would like to get this fix merged.

**Comment by trusktr** [comment]
In general, should we add tests for the examples?

**Pull request merged**